### DISCUSSION TITLE
Cannot tap linux-noah/noah: invalid syntax in tap! PLS HELP :(

### DISCUSSION THREAD

**Discussion by impostazioni** [Homebrew/discussions#3193] [Casks]
Hello guys first time here, I was trying to install Noah (https://github.com/linux-noah/noah) through Home-brew however i get the following message 

✧ brew install linux-noah/noah/noah
==> Tapping linux-noah/noah
Cloning into '/opt/homebrew/Library/Taps/linux-noah/homebrew-noah'...
remote: Enumerating objects: 137, done.
remote: Total 137 (delta 0), reused 0 (delta 0), pack-reused 137
Receiving objects: 100% (137/137), 18.77 KiB | 1.34 MiB/s, done.
Resolving deltas: 100% (41/41), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/linux-noah/homebrew-noah/noah.rb
noah: undefined method `cellar' for #<BottleSpecification:0x000000013a16d808>
Error: Cannot tap linux-noah/noah: invalid syntax in tap!
✧ 

Homebrew is up to date and brew doctor comes with the following message: 

✧ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  sdl
✧ 

So yes if anyone could help great !!! Im running Monterey 12.3.1 on M1 mac !!

**Comment by gromgit** [comment]
The tap should be cloned properly; Homebrew is complaining that the formulae therein use obsoleted syntax (they're _four years old_, after all).

Try removing the bottle block (`bottle do ... end`) in `/opt/homebrew/Library/Taps/linux-noah/homebrew-noah/noah.rb`, then `brew install noah` to see if there's anything else you need to fix.

**Reply by impostazioni** [reply]
Ty for the help however the only files files that I have in the tap directory are 
.		..		homebrew

**Reply by gromgit** [reply]
Hmmm, looks like Homebrew cleans up b0rked taps. You'll have to do the following, since the tap you're looking at seems to be abandoned:
1. Fork the original tap (https://github.com/linux-noah/homebrew-noah) into your own account (https://github.com/banisters/homebrew-noah).
2. Delete the bottle block for both formulae in your fork.
3. Install from your fork: `brew install banisters/noah/noah`.

**Reply by impostazioni** [reply]
How can I delete the bottle block for both formulae in my fork?

**Reply by gromgit** [reply]
https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files

(You can ignore the instruction in the above help page to create a pull request, just commit directly to your `master` branch.)
